addons/crm/models/crm_team.py

```diff
@@ -25,6 +25,19 @@ class Team(models.Model):
         compute='_compute_opportunities',
         string='Amount of quotations to invoice', readonly=True)
     dashboard_graph_model = fields.Selection(selection_add=[('pipeline', 'Pipeline')])
+    dashboard_graph_period_pipeline = fields.Selection([
```

Comment:
I don't understand. One field should be used for one thing to model. You should find a way to have meaningfull labels and we should never duplicate fields just for some label purpose. What is it for ?